h2o-algos/src/main/java/hex/deeplearning/DeepLearningModel.java

```diff
@@ -32,11 +35,420 @@
  * a scoring history, as well as some helpers to indicate the progress
  */
 
-public class DeepLearningModel extends Model<DeepLearningModel,DeepLearningModel.DeepLearningParameters,DeepLearningModel.DeepLearningModelOutput> implements Model.DeepFeatures {
-  @Override public ToEigenVec getToEigenVec() {
+public class DeepLearningModel extends Model<DeepLearningModel, DeepLearningModel.DeepLearningParameters, DeepLearningModel.DeepLearningModelOutput>
+        implements Model.DeepFeatures, Model.Contributions {
+  @Override
+  public ToEigenVec getToEigenVec() {
     return LinearAlgebraUtils.toEigen;
   }
 
+  class DeepSHAPContributionsWithBackground extends ContributionsWithBackgroundFrameTask<DeepSHAPContributionsWithBackground> {
```

Comment:
Is it possible to move this task somewhere out of DeepLearningModel? Thanks.

Reply:
It should be. I'll try that.